October-2015-Improvements.md

```diff
@@ -1,61 +0,0 @@
-Our open source community has been hard at work throughout September on some major improvements. We're going to keep up this tempo, and do a special live stream for our first birthday (October 15, 2015). Here are the improvements we just pushed to production:

```

Comment:
It was a duplicated file, the first one was renamed to `FreeCodeCamp-Site-Improvements-August-2015.md`